### DISCUSSION TITLE
Where can we get SAS_KEY_VALUE

### DISCUSSION THREAD

**Discussion by thunder-007** [Azure/azure-service-bus-emulator-installer#37] [Q&A] [answered]
what is this SAS_KEY_VALUE mean what should I replace it with in the string "Endpoint=sb://localhost;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;"

**Comment by krishankumar95** [comment, marked as answer]
`"Endpoint=sb://localhost;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;"` can be used as is. 

SAS_KEY_VALUE itself is the dummy key.
